aware-tests/src/main/java/com/aware/tests/TestESM.java

```diff
@@ -22,9 +26,9 @@
 
     @Override
     public void test(Context context) {
-//        testESMS(context);
+        testESMS(context);
 //        trialESMS(context);
-        testFlow(context);
+//        testFlow(context);
     }
 
     private void testFlow(Context context) {
```

Comment:
These tests were for flow functionality. Create and use testNotifications() to not mix tests.